pkg/obfuscator/ip_test.go

```diff
@@ -207,6 +206,14 @@ func TestIPObfuscatorConsistent(t *testing.T) {
 				"192.168.1.34": "x-ipv4-000010-x",
 			},
 		},
+		{
+			name:   "standard ipv4 with colons and standard ipv4 with dashes between should map to the same obfuscation value",
+			input:  []string{"obfuscate 10.0.129.220 and 10-0-129-220"},
+			output: []string{"obfuscate x-ipv4-000001-x and x-ipv4-000001-x"},
+			report: map[string]string{
+				"10.0.129.220": "x-ipv4-000001-x",
```

Comment:
thanks again for catching that one, that even went pass my end2end tests the other day